bertopic/_utils.py

```diff
@@ -147,3 +147,29 @@ def validate_distance_matrix(X, n_samples):
         raise ValueError("Distance matrix cannot contain negative values.")
 
     return X
+
+
+def get_unique_distances(dists: np.array, noise_min=1e-3, noise_max=1e-1) -> np.array:
+    """Check if the consecutive elements in the distance array are the same. If so, a small noise
+    is added to one of the elements to make sure that the array does not contain duplicates.
+
+    Arguments:
+        dists: distance array.
+        noise_min: the minimal magnitude of noise to be added.
+        noise_max: the maximal magnitude of noise to be added.
+
+    Returns:
+         Unique distances sorted in the preserved increasing order.
+
+    Raises:
+          ValueError: If the distance array is not sorted in the increasing order.
+    """
+
+    if not np.all(np.diff(dists) >= 0):
+        raise ValueError("The distances must be sorted in the increasing order")
```

Comment:
When did you run into this issue that this is needed? I can imagine that even if the value error is given, this should not be the responsibility of the user, or should it?

Reply:
I described the use case when this happens in the issue https://github.com/MaartenGr/BERTopic/issues/1907. 

I wanted to include this check as the assertion that the distances are originally sorted, otherwise the `get_unique_distances` would not work. if the distances are from linkage Z matrix, the order is always sorted.

What do you think about it?

Reply:
I'm not sure I understand. Issue #1907 describes when topics have the same distance not necessarily that the order of distances is descending instead of ascending. 

> I wanted to include this check as the assertion that the distances are originally sorted, otherwise the get_unique_distances would not work. if the distances are from linkage Z matrix, the order is always sorted.

Checks are always great but does it actually happen that the distances are not sorted? If yes, then how should the user use the error message to resolve the issue?